utils.go

```diff
@@ -4,7 +4,7 @@
  * Copyright 2019-Present Datadog, Inc.
  */
 
-package datadog
+package client
```

Comment:
Is this file generated?

Reply:
No. I moved it out as there were only static bits.